src/qt/bitcoingui.cpp

```diff
@@ -351,6 +354,11 @@ void BitcoinGUI::createActions()
     // initially disable the debug window menu item
     openRPCConsoleAction->setEnabled(false);
 
+    showMempoolStatsAction = new QAction(platformStyle->TextColorIcon(":/icons/mempoolstats"), tr("&Mempool Statistics"), this);
```

Comment:
Please remember to keep in mind licensing. typicons are CC-BY-3.0, which is a non-free license.